tests/fast/CoordinatorsForce.toml

```diff
@@ -0,0 +1,27 @@
+[[test]]
+testTitle = 'ForceMixedCoordinators'
+
+    [[test.workload]]
+    testName = 'Attrition'
+    machinesToKill = 10
+    machinesToLeave = 3
+    reboot = true
+    testDuration = 30.0
+
+    [[test.workload]]
+    testName = 'Attrition'
+    machinesToKill = 10
+    machinesToLeave = 3
+    reboot = true
+    testDuration = 30.0
+
+    [[test.workload]]
+    testName = 'Attrition'
+    machinesToKill = 10
+    machinesToLeave = 3
+    reboot = true
+    testDuration = 30.0
+
+    [[test.workload]]
+    testName = 'CoordinatorsForce'
```

Comment:
I think you need to add `testDelay = 20.0` or something here, so that attrition would have a chance to kill some coordinators before you attempt the coordinator change?

Reply:
By default, the `testDelay` parameter is 30s already